### ISSUE TITLE
Release and break times

### ISSUE THREAD

**Issue by vjeux** [codecat/tm-dashboard#15]
When someone releases it’s a super important event but right now the only way it manifests is that the arrow flashes for a split second. Would be great to add some kind of “toast” that says “Released for 200ms” this way it’s easier for the viewers to see that the streamer did release. Otherwise it’s not immediately obvious.

**Comment by codecat** [comment]
Good idea! Not sure how I would display this though. Don't want it to get in the way too much. (Plus there should be a setting to turn this on explicitly.)

**Comment by skybaks** [comment]
One possible idea for this could be to add some kind of animation to fade out a key upon release such as https://gfycat.com/smugdizzygrayfox
This would only help the case where a key quickly transitions from unpressed -> pressed -> unpressed, though
